configure

```diff
@@ -856,6 +856,18 @@ else
   leave 1
 fi
 
+# Check for -fno-semantic-interposition compiler support
+echo "" > test.c
+  cat > $test.c <<EOF
+int main() { return 0; }
+EOF
+if test "$gcc" -eq 1 && ($cc $CFLAGS -fno-semantic-interposition -c $test.c) >> configure.log 2>&1; then
+  echo "Checking for -no-semantic-interposition... Yes." | tee -a configure.log
+  SFLAGS="${SFLAGS} -fno-semantic-interposition"
```

Comment:
Should this be `${SFLAGS}` or `$SFLAGS`. The check for hidden attribute below uses `$SFLAGS`. Maybe it doesn't matter?

Reply:
I got that from the tests above this one actually. It doesn't really matter, but I'll remove the curly braces for simplicity.
Ref: https://unix.stackexchange.com/questions/4899/var-vs-var-and-to-quote-or-not-to-quote

Reply:
Fixed